includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -57,30 +121,59 @@ public function get_parent_id() {
 	}
 
 	/**
-	 * Gets the task content.
-	 *
-	 * @return string
-	 */
-	public function get_content() {
-		return '';
-	}
-
-	/**
-	 * Get the additional info.
+	 * Gets the task subtitle.
 	 *
 	 * @return string
 	 */
 	public function get_additional_info() {
-		return __( 'View and respond', 'woocommerce-payments' );
-	}
+		$disputes_due_within_7d = $this->get_disputes_needing_response_within_days( 7 );
+		$disputes_due_within_1d = $this->get_disputes_needing_response_within_days( 1 );
+
+		if ( count( $disputes_due_within_7d ) === 1 ) {
+			$local_timezone    = new \DateTimeZone( wp_timezone_string() );
+			$dispute           = $disputes_due_within_7d[0];
+			$due_by_local_time = ( new \DateTime( $dispute['due_by'] ) )->setTimezone( $local_timezone );
+
+			if ( count( $disputes_due_within_1d ) > 0 ) {
+				return sprintf(
+					/* translators: %s is time, eg: 11:59 PM */
+					__( 'Respond today by %s', 'woocommerce-payments' ),
+					$due_by_local_time->format( 'g:i A' )
```

Comment:
this should be formatted using the `time_format` option. Likely wrapped with `wc_format_datetime`

Reply:
Ah, TIL! 

I'm thinking that these formats on the Overview task will have to be changes to use the `time_format`/`date_format` options as well:

https://github.com/Automattic/woocommerce-payments/blob/ff1996b27bbcdf5d7dd24bea9daf871738d1e62d/client/overview/task-list/dispute-tasks.ts#L153-L172

Reply:
Updated in f4bb038. I'll look at incorporating the store's formatting for the Overview task as well.